### DISCUSSION TITLE
Problem with Freehand digitizing

### DISCUSSION THREAD

**Discussion by Zou48** [opengisch/QField#5794] [Bugs]
### Describe the Bug

When I use QFIELD on my tablet with a stylus or tablet pen and want to use the 'Freehand digitizing' mode, it doesn't work,. I can only make straight lines or polygons even if the mode is activate. The 'Freehand digitizing' mode only works with my mouse connected to my tablet.
Reproduction steps

### Reproduction steps
    OPEN QfieldProject
    Select a polygon or line layer
    Click on digitize mode
    Click on Freehand digitize mode
    Use the pen or stylus
    Surrend15

### Expected behavior
Freehand drawing

### Observed behavior
Straight line or polygon drawn

![Screenshot_20241107-171641](https://github.com/user-attachments/assets/5965d3c7-48ba-49ab-add4-79c64425483a)

### QGIS
    OS: [Windows]
    QGIS Version [e.g. 3.34.0]
    QFieldSync Version [e.g.4.0.0]

### QField
    Device: [e.g. BlackView Active 8 Pro]
    OS: [e.g. Android 13]
    QField version: 3.4.4 - Ebo (5cc32f) Qt 6.7.3

### Additional information

    Problem started happening recently, didn't happen in an older version of QField: No, I downloaded old version too
    Problem can be reliably reproduced, doesn't happen randomly: Yes
    Problem happens with all files and projects, not only some files or projects: Yes

![ezgif-1-bc8ef44aed](https://github.com/user-attachments/assets/962eaa16-3d6e-4ae7-9936-71602111e698)

**Comment by nirvn** [comment]
Closing in favor of the issue filed here: https://github.com/opengisch/QField/issues/5793
